server/src/service/CardService.js

```diff
@@ -131,4 +131,36 @@ export class CardService extends BaseService {
 
         await this.cardRepository.save(card);
     }
+
+    @Transactional()
+    async addMemberToCardByUserIds({ cardId, userId, userIds }) {
+        const card = await this.cardRepository.findOne(cardId);
+
+        if (!card) throw new EntityNotFoundError('card is not exist');
+
+        const { id: boardId } = await this.customBoardRepository.findBoardByCardId(cardId);
+        const isExistUser = await this.customBoardRepository.existUserByBoardId({
+            boardId,
+            userId,
+        });
+
+        if (userIds.length === 0) {
+            await this.memberRepository.delete({ card: cardId });
+            return;
+        }
+
+        if (!isExistUser) throw new ForbiddenError('no access to this card');
```

Comment:
```suggestion
        if (!isExistUser) throw new ForbiddenError('no access to this card');
        
        if (userIds.length === 0) {
            await this.memberRepository.delete({ card: cardId });
            return;
        }
```
두 개의 로직이 순서가 바뀌어야 된다고 생각합니당. 권한 검사를 하고 나서 수정해야 할듯 해요

Reply:
commit 770c177
수정했습니다!